web/models/package.ex

```diff
@@ -131,6 +131,12 @@ defmodule HexWeb.Package do
     |> select([o], count(o.id) == 1)
   end
 
+  def all_has_docs do
+    from( p in Package,
+         join: r in assoc(p, :releases),
+         where: r.has_docs == true)
+  end
```

Comment:
This will generate some duplicate package names since a package can have multiple releases.